setup/steps/09_deploy_via_modelservice.sh

```diff
@@ -26,9 +26,12 @@ if [[ $LLMDBENCH_CONTROL_ENVIRONMENT_TYPE_MODELSERVICE_ACTIVE -eq 1 ]]; then
     export LLMDBENCH_DEPLOY_CURRENT_MODEL_ID=$(model_attribute $model modelid)
     export LLMDBENCH_DEPLOY_CURRENT_MODEL_ID_LABEL=$(model_attribute $model modelid_label)
 
-    # If LLMDBENCH_VLLM_MODELSERVICE_URI is not defined, set it to pvc://
-    if [[ -z "$LLMDBENCH_VLLM_MODELSERVICE_URI" ]]; then
+    if [[ $LLMDBENCH_VLLM_MODELSERVICE_URI_PROTOCOL == "pvc" || ${LLMDBENCH_CONTROL_ENVIRONMENT_TYPE_STANDALONE_ACTIVE} -eq 1 ]]; then
       export LLMDBENCH_VLLM_MODELSERVICE_URI="pvc://${LLMDBENCH_VLLM_COMMON_PVC_NAME}/models/$(model_attribute $model model)"
+      mount_model_volume=true
+    else
+      export LLMDBENCH_VLLM_MODELSERVICE_URI="hf://$(model_attribute $model model)"
+      mount_model_volume=false
```

Comment:
Hmm even for the `hf` case, mount_model_volume can be true, which means creating an emptyDir volume with size = `LLMDBENCH_VLLM_COMMON_PVC_MODEL_CACHE_SIZE` and mounting this volume to the vLLM container. I think we still want this to be true.